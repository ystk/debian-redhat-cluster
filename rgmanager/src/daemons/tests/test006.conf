<?xml version="1.0"?>
<!-- The IP address should have a NEEDSTOP in the old tree and NEEDSTART
     in the new tree, -->
<cluster>
<rm>
	<resources>
    		<service name="test1"/>
		<script name="initscript" file="/etc/init.d/sshd"/>
		<ip address="192.168.1.2" monitor_link="yes"/>
	</resources>
	<service ref="test1">
		<ip ref="192.168.1.2"/>
		<script ref="initscript"/>
	</service>
</rm>
</cluster>
