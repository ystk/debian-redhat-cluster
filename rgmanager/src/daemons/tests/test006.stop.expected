Stopping test1...
[stop] script:initscript
[stop] ip:192.168.1.2
[stop] service:test1
Stop of test1 complete
